frontend/src/lib/lemon-ui/ProfilePicture/ProfilePicture.scss

```diff
@@ -40,6 +40,13 @@
         font-size: 20px;
     }
 
+    &.lg {
+        width: 32px;
+        height: 32px;
+        line-height: 32px;
+        font-size: 16px;
+    }
```

Comment:
I can see how this is not clear, but the _default_ size already has the same dimensions as this new `lg`, just with a slightly larger font. So now we basically have two "large" sizes, and it's hard to say which to use when. I propose we remove `width`, `height`, `line-height` and `font-size` from the base class, and `.profile-picture`, and enforce there always being a size in `ProfilePicture` with `size = 'lg'`